tutoroars/templates/oars/apps/data/assets/datasets/OpenedX_Clickhouse/xapi_events_all_parsed.yaml

```diff
@@ -0,0 +1,186 @@
+{% raw %}
```

Comment:
Originally, we needed these `{% raw %} {% endraw %}` so that jinja filters weren't expanded by Tutor during init.

But since the row-level security seems to mean that we don't need to embed our `can_view_course` filter logic in the queries themselves, then we probably don't need these `{% raw %} {% endraw %}` statements bookending all of these yaml files anymore either.

But if we do, then we should write a make script for adding them to an export zip.

Reply:
I'm not sure we'll be able to use ClickHouse row level security without giving each user a ClickHouse user, which is probably prohibitively complicated to manage, but I'm open to suggestions there.

Reply:
[Superset supports Row Level Security](https://superset.apache.org/docs/security/) :)
I can attach a RowLevelSecurityFilter to the Clickhouse xapi dataset and filter based on the `course_id`s the user is allowed to see. I've been able to do this manually, just have to figure out how to add/update the filter programmatically, since there isn't an import/export or command-line API for it.